mmv1/third_party/terraform/services/memorystore/resource_memorystore_instance_test.go

```diff
@@ -1537,3 +1537,74 @@ data "google_project" "project" {
 }
 `, context)
 }
+
+func TestAccMemorystoreInstance_memorystorePscAutoInstanceClusterDisabled(t *testing.T) {
```

Comment:
could you add a couple more test steps here so that the test flow is:

1. setting both
2. setting just one
3. setting just the other
4. setting neither